clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/loadbalancer/ops/UpsertAzureLoadBalancerAtomicOperation.groovy

```diff
@@ -51,25 +50,40 @@ class UpsertAzureLoadBalancerAtomicOperation implements AtomicOperation<Map> {
     task.updateStatus BASE_PHASE, "Initializing upsert of load balancer $description.loadBalancerName " +
       "in $description.region..."
 
+    Map<String, Boolean> resourceCompletedState = new HashMap<String, Boolean>()
+
     try {
-      DeploymentExtended deployment = description.credentials.resourceManagerClient.createLoadBalancerFromTemplate(description.credentials,
+
+      task.updateStatus(BASE_PHASE, "Beginning load balancer deployment")
+
+      String resourceGroupName = AzureUtilities.getResourceGroupName(description.appName, description.region)
```

Comment:
I see that further down, we have to pass the resource group name to subsequent calls in this method and unfortunately, there isn't a mechanism to return the resource group name from the call to CreateResourceFromTemplate. For this PR I wouldn't make the change I suggested above but this is definitely something that needs to be addressed as we move forward with creating other resources.